java-compat/test/android/controller/build.gradle

```diff
@@ -10,7 +10,7 @@ buildscript {
         }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.4.2'
+        classpath 'com.android.tools.build:gradle:8.1.1'
```

Comment:
I don't follow: the distributionUrl is gradle-8.3 but the classpath is 8.1.1? Why this discrepancy?

Reply:
They are different packages, one is Gradle and the other is the Android plug-in for Gradle